Jenkinsfile

```diff
@@ -100,11 +100,16 @@ def runRelease(architectures) {
                 publishChocoPackage(version, jfrogCliRepoDir, architectures)
             }
         } else if ("$EXECUTION_MODE".toString().equals("Build CLI")) {
+            stage("Audit") {
```

Comment:
This will run twice, once for each executable name.
Might want to add with an if condition such as `if (identifier == "v2-jf") {}`